resources/assets/scripts/app.js

```diff
@@ -3,27 +3,28 @@
  */
 import 'jquery';
 import 'bootstrap';
+import { router } from 'js-dom-router';
 
 /**
- * Local Dependencies
+ * DOM-based routing
  */
-import Router from './util/Router';
-import common from './routes/common';
-import aboutUs from './routes/about';
-import { ready } from './utils';
+import about from './routes/about';
 
 /**
- * Populate the Router instance with DOM routes.
+ * Below is an example of a dynamic import; it will not be loaded until it's needed.
  *
- * common – Fired on all pages.
- * aboutUs – Fired on the About Us page, note the change from about-us to aboutUs (camelCase).
+ * See: {@link https://webpack.js.org/guides/code-splitting/#dynamic-imports | Dynamic Imports}
  */
-const routes = new Router({
-  common,
-  aboutUs,
-});
+const home = async () => import(/* webpackChunkName: "home-page" */ './routes/home');
```

Comment:
```suggestion
const home = async () => import(/* webpackChunkName: "scripts/home-page" */ './routes/home');
```

This would put the chunk in the scripts folder. Otherwise it all looks/works good.

Reply:
`scripts/` or `scripts/routes/`? 🤔

Reply:
Good point; the latter.